Source/WebCore/css/StyleProperties.cpp

```diff
@@ -1355,6 +1354,38 @@ String StyleProperties::pageBreakPropertyValue(const StylePropertyShorthand& sho
     }
 }
 
+String StyleProperties::columnBreakPropertyValue(const StylePropertyShorthand& shorthand) const
+{
+    ASSERT(shorthand.length() == 1);
+    auto value = getPropertyCSSValue(shorthand.properties()[0]);
+    if (!value)
+        return String();
+    // FIXME: Remove this isCSSWideKeyword check after we do this consistently for all shorthands in getPropertyValue.
+    if (value->isCSSWideKeyword())
+        return value->cssText();
+
+    if (!is<CSSPrimitiveValue>(*value))
+        return String();
+
+    CSSValueID valueId = downcast<CSSPrimitiveValue>(*value).valueID();
```

Comment:
```suggestion
    auto valueID = propertyAsValueID(shorthand.properties()[0]);
    if (!valueID)
        return emptyString();
    // FIXME: Remove this isCSSWideKeyword check after we do this consistently for all shorthands in getPropertyValue.
    if (isCSSWideKeyword(*valueID))
        return nameString(*valueID);
```

This can be shorter with `propertyAsValueID`.